src/runtime/particle-spec.ts

```diff
@@ -367,23 +357,25 @@ export class ParticleSpec {
     return this.toString();
   }
 
-  private validateTrustClaims(claims: SerializedParticleTrustClaimSpec[]): Map<string, string> {
-    const results: Map<string, string> = new Map();
+  private validateTrustClaims(claims: ParticleTrustClaim[]): Map<string, ParticleTrustClaim> {
+    const results: Map<string, ParticleTrustClaim> = new Map();
     if (claims) {
       claims.forEach(claim => {
+        assert(!results.has(claim.handle), `Can't make multiple claims on the same output (${claim.handle}).`);
```

Comment:
Shouldn't these be exceptions rather than asserts, per our previous conversation? Invalid input should not make the code crash.

Reply:
Converted these to throw exceptions too